### PR TITLE
Updated documentation for solving building error

### PR DESCRIPTION
**Pull request by stepski011**
Updated readme file, adding the solution to error which occurs while building AF. Testing was provided on virtual machines as well as under the native Linux administration. Script fails to fetch .jar file for which JDK is required. Not every system have JDK installed by default.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,19 +42,21 @@ git submodule update
 
 To build the 5GMS Application Function from the source:
 
-```bash
+```
 cd ~/rt-5gms-application-function
-meson build
+meson build --prefix=`pwd`/install
```
Comment (davidjwbbc) [inline README.md:47]: Please do not revert these changes.

Please rebase your changes onto the latest HEAD on the `development` branch.

File: README.md [hunk 2]
```diff
@@ -42,19 +42,21 @@ git submodule update
...
 ninja -C build
 ```
+In case of error during building process using `meson`, please check your Java version.
+Error is probably related to fetching `openapi-generator-cli.jar` module, which requires [Java Development Kit](https://www.oracle.com/java/technologies/downloads/) installed and configured on your system.
```
Comment (davidjwbbc) [inline README.md:51]: The Ubuntu package `default-jdk` or `java-latest-openjdk` for RedHat like distributions is sufficient here.

This will only impact the ability to run the `openapi-generator-cli.jar` file, if there is a problem fetching the jar file then that will be a problem with network access or DNS.

File: README.md [hunk 3]
```diff
@@ -42,19 +42,21 @@ git submodule update
...
 
 ## Installing
 
 To install the built Application Function as a system process:
 
 ```bash
 cd ~/rt-5gms-application-function/build
-sudo meson install --no-rebuild
+ninja install
```
Comment (davidjwbbc) [inline README.md:59]: Do not revert these changes either.

**Review by davidjwbbc — CHANGES_REQUESTED** [review]
Original PR includes reverts for previous documentation changes that should not be here. The commits should have been rebased against the HEAD of the `development` branch.

File: README.md (hunk 2 above)
Comment (stepski011) [reply README.md:51]: Every error on Debian based machines was related to executing .jar file. One issue was raised on that too. It's true that it shouldn't always be the case. The pull request can be closed since that was only modification, and others aren't supposed to be reverted.

**Comment by davidjwbbc** [comment]
I've incorporated a similar statement in PR #44 along with a few other documentation changes. I think this addresses the issue in this PR so I'll close this one.
